tests/10apidoc/09synced.pl

```diff
@@ -205,7 +205,34 @@ sub await_sync_presence_contains {
 }
 
 
-push @EXPORT, qw( assert_room_members assert_state_room_members_matches );
+push @EXPORT, qw( assert_room_members assert_state_room_members_matches assert_room_state);
+
+# assert that the state body of a sync response is made up of the given state types.
+sub assert_room_state {
+   my ( $body, $room_id, $state_types ) = @_;
```

Comment:
some docs on what state_types should look like would be helpful, to save people reverse-engineering it.